### DISCUSSION TITLE
Progressive Hydration

### DISCUSSION THREAD

**Discussion by nesbtesh** [jaredpalmer/razzle#1445] [Q&A]
Has anyone managed to do progressive hydration?

**Comment by fivethreeo** [comment]
You mean suspense?

**Reply by nesbtesh** [reply]
No, partial hydration is for optimizing the first render of the App, the idea is to only hydrate the part of the App that uses Javascript. What I have managed to do so far is to create a component that doesn't hydrate the html if the component is static. I am working now on creating a component that only hydrates the component if its in the view. Using the Intersection Observer.

When this is done I should manage to have a Web Performance of 100% with a first render of less than 1s

**Reply by fivethreeo** [reply]
Can you answer this one when you are done? 😉
